public/javascripts/items.js

```diff
@@ -39,9 +39,8 @@ const clickToUpdate = parentClass => event => {
 const titleEdited = event => {
   const elementToHide = $( event.target )
   const id = elementToHide.data( 'id' )
-  const elementToShow = $( selector( 'title', id, 'span' ) )
-
-  if( event.charCode === RETURN_KEY ) {
+  const elementToShow = $( selector( 'item__title', id, 'span' ) )
+  if( event.keyCode === RETURN_KEY ) {
```

Comment:
Please add whitespace back - there *should* be whitespace between the setup of a function, and the logic.